### PR TITLE
fix(stats): include abuse outcomes in total dropped

### PR DESCRIPTION
**Pull request by cathteng**
It has been reported many times that the numbers on the stats page don't line up. Support has done some digging and it appears that this is due to abuse outcomes not being included in the total count of dropped events. This has been fixed here.

### CODE REVIEW

**Review by leedongwei — APPROVED** [review]

**Review by nhsiehgit — CHANGES_REQUESTED** [review]
going to block on this until we have a clear outcome from product 😬

**Comment by Dhrumil-Sentry** [comment]
Let's discuss this a bit more before we merge any changes here

**Comment by JoshFerge** [comment]
what did we end up deciding here?

**Comment by cathteng** [comment]
@JoshFerge i can follow up

**Comment by cathteng** [comment]
parking this for now

**Pull request closed without merging**
